### PR TITLE
add Message to wrap all.

### PR DESCRIPTION
**Pull request by siddontang**
Later we will use one listener in TiKV, so we must unity all messages into one Message first. 

@ngaut @qiuyesuifeng @disksing @morefreeze @BusyJay 

This will affect tikv + pd + ticlient. I will first try to refactor tikv, then @morefreeze and @qiuyesuifeng  can update ticlient and pd.

### CODE REVIEW

File: proto/msgpb.proto [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    Cmd          = 1;
+    CmdResp      = 2;
+    Raft         = 3;
+    KvReq        = 4;
+    KvResp       = 5;
+}
+
+// Message holds all messages communicating with TiKV.
+message Message {
+    optional MessageType                 msg_type  = 1;
```
Comment (morefreeze) [inline proto/msgpb.proto]: I think explicit `default=None` is better.

Comment (siddontang) [inline proto/msgpb.proto]: no need. The default is the first item in enum .

**Comment by ngaut** [comment]
LGTM

**Comment by qiuyesuifeng** [comment]
LGTM

**Pull request merged**